scripts/shared/deploy.sh

```diff
@@ -45,12 +45,16 @@ import_image quay.io/submariner/submariner-route-agent
 load_deploytool $deploytool
 deploytool_prereqs
 
-run_parallel "{1..3}" prepare_cluster "$SUBM_NS"
+run_parallel "${clusters[*]}" prepare_cluster "$SUBM_NS"
 
-with_context cluster1 setup_broker
+with_context "${clusters[0]}" setup_broker
 install_subm_all_clusters
 
 deploytool_postreqs
 
-with_context cluster2 connectivity_tests
+if [ "${#clusters[@]}" -gt 3 ]; then
```

Comment:
```suggestion
if [ "${#clusters[@]}" -gt 2 ]; then
```

Otherwise even with 3 clusters it wont run it

Reply:
yeah I meant 2 😄